contrib/test.sh

```diff
@@ -20,12 +20,12 @@ then
     cargo fmt -- --check
 fi
 
-# Pin dependencies required to build with Rust 1.48.0
-if cargo --version | grep "1\.48\.0"; then
+# Pin dependencies required to build with Rust 1.56.1
+if cargo --version | grep "1\.56\.1"; then
     cargo update -p once_cell --precise 1.13.1
-    cargo update -p quote --precise 1.0.28
-    cargo update -p syn --precise 2.0.32
-    cargo update -p proc-macro2 --precise 1.0.63
+    cargo update -p quote --precise 1.0.35
+    cargo update -p syn --precise 2.0.46
+    cargo update -p proc-macro2 --precise 1.0.74
```

Comment:
In ad446c23275cf2c97e993102014c8498c69f01f3:

Are these pins actually necessary?

Can you try removing the pins, compiling with 1.56.1, and then re-adding them as necessary?

Reply:
You were right. They are not necessary. I've removed them in c301ce3688e66a1f0f041154c4f2455c9dfa10e4.